.github/workflows/test_flavor_cloud_ng.yml

```diff
@@ -41,6 +41,7 @@ jobs:
   test:
     name: Test-NG flavor in Cloud ${{ inputs.flavor }} (${{ inputs.arch }})
     runs-on: "ubuntu-24.04"
+    if: ${{ inputs.flavor != 'skip' }}
```

Comment:
I guess these if-conditions can be removed. See .github/workflows/tests.yml comment about empty matrices.